indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -128,8 +135,19 @@ def _process(self):
         args = parser.parse({
             'files': EditingFilesField(self.event, self.contrib, self.editable_type, required=True)
         })
-
-        create_new_editable(self.contrib, self.editable_type, session.user, args['files'])
+        service_url = editing_settings.get(self.event, 'service_url')
+        initial_state = InitialRevisionState.new if service_url else InitialRevisionState.ready_for_review
+
+        editable = create_new_editable(self.contrib, self.editable_type, session.user, args['files'], initial_state)
+        db.session.commit()
+        if service_url:
+            try:
+                service_handle_editable(editable)
+            except ServiceRequestFailed as exc:
+                delete_editable(editable)
+                db.session.commit()
+                logger.exception('Failed to contact service. Got: %r', exc.message)
```

Comment:
No need to include exception details again; `logger.exception()` already adds the full exception and traceback to the logging record.